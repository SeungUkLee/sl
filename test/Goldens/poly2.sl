(* polymorphic *)

let foo f g = fun x y -> f (g x) (g y) in foo
