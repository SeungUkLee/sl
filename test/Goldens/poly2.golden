"<<function>> : (p -> p -> o) -> (q -> p) -> q -> q -> o"